modules/mesos/pom.xml

```diff
@@ -20,7 +20,8 @@
 <!--
     POM file.
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Do you have to change XML header?

Reply:
@alamar No, but Intellij simplified it and it works. Do you think I have to revert it?

Reply:
I'd revert it since all other pom.xmls seem to have same header line.

Reply:
@alamar Agree. Done.